src/combinatorics/simple_graph/connectivity.lean

```diff
@@ -1872,4 +1879,128 @@ sym2.ind (λ v w, is_bridge_iff_adj_and_forall_cycle_not_mem) e
 
 end bridge_edges
 
+section induced
+
+/-! ### Connectivity of induced graphs -/
+
+lemma induce_singleton_connected (v : V) :
+  (G.induce {v}).connected :=
+begin
+  rw connected_iff,
+  refine ⟨_, by simp⟩,
+  rintro ⟨x, hx⟩ ⟨y, hy⟩,
+  rw set.mem_singleton_iff at hx hy,
+  subst_vars,
+end
+
+lemma induce_union_connected {H K : set V}
```

Comment:
It's confusing to call bare sets `H` and `K`
```suggestion
lemma induce_union_connected {s t : set V}
```

Reply:
Ah, yeah, sorry, remnant of a different notation!